src/main/java/com/hackaton/gui/Main.java

```diff
@@ -0,0 +1,45 @@
+package com.hackaton.gui;
+
+import java.awt.*;
+import java.io.File;
+
+public class Main implements Runnable {
+
+    private ResultWindow result;
+    private MainWindow mainWindow;
+
+    /**
+     * When an object implementing interface <code>Runnable</code> is used
+     * to create a thread, starting the thread causes the object's
+     * <code>run</code> method to be called in that separately executing
+     * thread.
+     * <p>
+     * The general contract of the method <code>run</code> is that it may
+     * take any action whatsoever.
+     *
+     * @see Thread#run()
+     */
+    @Override
+    public void run() {
+        mainWindow = new MainWindow(this);
+        result = new ResultWindow(this);
+        showMain();
+    }
+
+    void showResult(){
+        File file = mainWindow.getFile();
```

Comment:
Tutaj dostajemy plik wybrany przez użyszkodnika. Należy na nim pracować i jako List przekazać dane do parametru setModel.

Reply:
W javie nie powinno używać się konstrukcji this.pole?

Reply:
Chodzi ci czemu to nie jest pole klasy? Bo jeszcze nie ma co z tym zrobić. Nie wiem co sakso będzie chciał. Czy pytasz czemu jest getter zamiast pola? Enkapsulacja.